src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.cs

```diff
@@ -1526,9 +1559,15 @@ ProviderTypes.Twitter     or ProviderTypes.Weibo         or ProviderTypes.Zoom
                 ProviderTypes.Vimeo => (string?) context.UserInfoResponse?["uri"] is string uri &&
                     uri.StartsWith("/users/", StringComparison.Ordinal) ? uri["/users/".Length..] : null,
 
+                // VK ID returns the user identifier as a custom "user/user_id" node:
+                ProviderTypes.VkId => (string?) context.UserInfoResponse?["user"]?["user_id"],
+
                 // WordPress returns the user identifier as a custom "ID" node:
                 ProviderTypes.WordPress => (string?) context.UserInfoResponse?["ID"],
 
+                // Yandex returns the user identifier as a custom "id" node:
+                ProviderTypes.Yandex => (string?) context.UserInfoResponse?["id"],
```

Comment:
There's already a case for `id` earlier in that handler so you can add `Yandex` there.